lib/iev/termbase/terminology_sheet.rb

```diff
@@ -1,108 +1,105 @@
 require_relative "metadata_section"
 require_relative "terms_section"
 
-module Tc211::Termbase
+module Iev::Termbase
```

Comment:
`TerminologySheet` was used to parse ISO TC 211 excel and is not relevant here. In ISO TC 211 excel, every language is on a separate sheet, and each sheet contained a separate metadata and terms section.

All language terms on IEV excel are on the same sheet, and there is no metadata section to parse.